bouncycastle/src/main/scala/tsec/Bouncy.scala

```diff
@@ -0,0 +1,16 @@
+package tsec
+
+import java.security.Security
+
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+
+trait Bouncy
```

Comment:
shouldn't this be _sealed_ ?

Reply:
Can be, sure

Reply:
👍 since it's a tag there's no reason to let ~~an attacker~~ some lazy fuck forge one